internal/pkg/service/buffer/storage/level/local/writer/volume/volume.go

```diff
@@ -243,18 +240,6 @@ func (v *Volume) checkDrainFile() error {
 	return nil
 }
 
-func (v *Volume) openedWriters() (out []writer.Writer) {
-	v.writersLock.Lock()
```

Comment:
Presunul som tuto metodu z `volume.go` do `writer.go`, kde je suvisiaci kod + je uz public ako `Writers`, budem to potrebovat v dalsom PR (pre ziskavanie statistik z jednotlivych writers).